docs/function-pointers-as-arguments.md

```diff
@@ -425,7 +425,7 @@ end program prog_prog_boo
 
 If you face problems with Pyccel, please take the following steps:
 
-1.  Consult our documentation in the tutorial directory;
+1.  Consult our documentation in the docs directory;
 2.  Send an email message to pyccel@googlegroups.com;
```

Comment:
Is this suggestion still pertinent? I think the email still exists but I'm not sure it's the right thing to use

Reply:
I think only one person has ever used it 🤣